include/dlpack/dlpack.h

```diff
@@ -172,7 +178,7 @@ typedef struct {
   /*! \brief The device of the tensor */
   DLDevice device;
   /*! \brief Number of dimensions */
-  int ndim;
+  int32_t ndim;
```

Comment:
Could consider using `DLPACK_INT32_ENUM_TYPE` here to avoid future divergence.

Reply:
I believe we want to update [Lines 90/96](https://github.com/dmlc/dlpack/pull/86/files#diff-1564e5abd0da5cf46e82b28b0058996df944007494520a2b0a6b629880eaa097R96) as well.

`int device_id` -> `int32_t device_id` or equivalently 
-> `DLPACK_INT32_ENUM_TYPE device_id` as mentioned above.

Reply:
in this case this is not an enum, mainly to clarify. `DLPACK_INT32_ENUM_TYPE` is indeed confusing, going to change `DLPACK_INT32_ENUM_ANNOTATION`

Reply:
In that case, I think the only change we would want is `int device_id -> int32_t device_id`.